pkg/jobs/registry.go

```diff
@@ -292,6 +300,13 @@ func (r *Registry) ID() base.SQLInstanceID {
 	return r.nodeID.SQLInstanceID()
 }
 
+// StandaloneSQLInstance returns true if the registry is running within a
```

Comment:
nit: this doesn't really feel like it belongs on jobs.Registry to me. I think you can get this bool in your job off the existing `execCtx` argument, e.g. `execCtx.ExecCfg().NodeInfo.NodeID.OptionalNodeID()`